_posts/2015-12-05-grip-it-and-rip-it.md

```diff
@@ -0,0 +1,53 @@
+
+Considering SailsJS (Node) as a backend, it only made sense to use JavaScript on the front-end, especially since we needed to incorporate SocketIO.  In the end, we went with AngularJS as it contained libraries for interfacing with Google Maps.
+
+The conclusion of the project was accomplished by ripping into the code and organically figuring out what would work and what wouldn't.  But there is one more reason you should just start coding.
+
+#### You Have Too Much of a Clue
+
+Sorry, I was too lazy to come up with a more catchy subtitle.  The reality is some of you have more experience in different areas and you know exactly what you need to do to complete a project.  You think about the architecture and map it out in your head.  You know the stack you want to use, and you put together the list of libraries that you need.  If you're like me, you've probably completed the project in your head and already have that feeling of accomplishment, but in reality you haven't started.  Yes, it's that feeling of, "I got this!" that paralyzes your progress.
+
+The truth of the matter is there's a term for people like you and me, and it's called "Analysis Paralysis."  You've planned out everything, but unfortunately, what's in your head has to make it into your text editor.  If you feel yourself over-thinking to the point of overwhelming, just open up your editor and begin coding.  Once you begin dumping what's in your head onto a working document, you'll be surprised as to how less complicated the project may be.  You may not need those libraries you thought about.
+
+Recently, I worked on a project that required a .NET integration from an open source PHP application.  I was thinking in my head about all of the pieces I needed in order to perform the integration.  In .NET, there's an MVC Framework configuration that allows you to quickly put together a product with minimal effort.  My experience as an API developer pointed me to using this MVC configuration; However, I ran into a game changer.  While building the API, I found that the framework has a configuration called Web API 2.  This configuration, by default, has a very similar infrastructure to that of Symfony (PHP framework) with entity relationships behind an MVC pattern.  Using this configuration saved me a significant amount of time and proved to be a better option to what I initially invisioned.  The reality is, doing the work led me to the right path in completing the work.  So just do it.
+
+#### Do It
+
+<img src="http://i.memecaptain.com/gend_images/JoT9Qw.png" height="250px" alt="Do It!" />
```

Comment:
yup plus @rortega1 be sure to use markdown for images... unless you need custom settings on html elements of course. but otherwise `![](image/url.png)` should work.